com.unity.shadergraph/Editor/GraphDeltaRegistry/Registry/GraphDeltaExtensions.cs

```diff
@@ -42,10 +42,18 @@ public static RegistryKey GetRegistryKey(this FieldHandler field)
             return field.GetMetadata<RegistryKey>(kRegistryKeyName);
         }
 
-        public static void AddReferenceNode(this GraphHandler handler, string nodeName, string referenceName, Registry.Registry registry)
+        public static void AddReferenceNode(this GraphHandler handler, string nodeName, string contextName, string contextEntryName, Registry.Registry registry)
```

Comment:
How do we want to handle specifying the name parameters? 

1. nodeName will be based on the guid of the variable node on the graph? (at least for now this is how `GraphDataNodeModel` does it)
2. contextName is the name of the property context node I assume
3. will contextEntryName be the name of the variable/property itself?

Reply:
Yes, yes, yes.

The name of a context node is strictly tied to the RegistryKey as they must be unique. We need to explore the Context Node API a lot more, but this is functional.